package.json

```diff
@@ -69,6 +69,7 @@
     "url": "https://github.com/elastic/kibana.git"
   },
   "resolutions": {
+    "@babel/types": "^7.15.6",
```

Comment:
Storybook was making this use `7.14.5`, which didn't make the types not compatible with https://github.com/elastic/kibana/blob/master/packages/kbn-dev-utils/src/vscode_config/update_vscode_config.ts#L228